reVX/wind_setbacks/wind_setbacks.py

```diff
@@ -0,0 +1,1386 @@
+# -*- coding: utf-8 -*-
+"""
+Handler to convert exclusion to/from .h5 and .geotiff
+"""
+from abc import ABC, abstractstaticmethod, abstractclassmethod
+from concurrent.futures import as_completed
+from earthpy import clip as cl
+import fiona
+import geopandas as gpd
+import h5py
+import json
+import logging
+import numpy as np
+import os
+from rasterio import features
+import re
+from shapely.geometry import shape
+from warnings import warn
+
+from rex.utilities import parse_table, SpawnProcessPool
+from reV.handlers.exclusions import ExclusionLayers
+
+logger = logging.getLogger(__name__)
+
+
+class BaseWindSetbacks(ABC):
+    """
+    Create exclusions layers for wind setbacks
+    """
+    MULTIPLIERS = {'high': 3, 'moderate': 1.1}
+
+    def __init__(self, excl_h5, hub_height, rotor_diameter, regs_fpath=None,
+                 multiplier=None, hsds=False, chunks=(128, 128)):
+        """
+        Parameters
+        ----------
+        excl_h5 : str
+            Path to .h5 file containing or to contain exclusion layers
```

Comment:
This is where the new setbacks will be written? This is not a source file, correct? If it is a source file, what is being used from it?